dockers/docker-lldp/lldpmgrd

```diff
@@ -198,6 +158,13 @@ class LldpManager(object):
         Subscribe to CONFIG_DB - get notified of port config changes
         Update LLDP configuration accordingly.
         """
+        self.log_info("Starting up...")
+
+        if not os.geteuid() == 0:
```

Comment:
Should it be part of the base class?
We can initialize the base class with parameters?

Reply:
This is a good suggestion for a future improvement. For now I'd like to transition daemons to inherit from the base class, then we can assess other remaining duplicated code (like this), and make improvements accordingly.